go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/konveyor/tackle2-seed
 
-go 1.19
+go 1.18
```

Comment:
Changed to 1.18 mainly because we're at 1.18 for the hub and other repositories.  As a result, I don't have 1.19 installed in my dev env.